.github/workflows/pip-requirements.yml

```
@@ -17,17 +17,12 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout PR target branch
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

```suggestion
        uses: actions/checkout@v4
```
no need to downgrade